### PR TITLE
en: add IAM ENV for backup and restore of s3

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add IAM ENV for import and export in backup.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs-tidb-operator/pull/40, https://github.com/pingcap/docs-tidb-operator/pull/57
- Other reference link(s):

### CODE REVIEW

**Comment by ran-huang** [comment]
An error in https://github.com/pingcap/docs-tidb-operator/pull/40 has been fixed in https://github.com/pingcap/docs-tidb-operator/pull/57. Please note that when reviewing this PR.

File: en/backup-to-s3.md [hunk 1]
```diff
@@ -10,113 +10,160 @@ This document describes how to back up the data of the TiDB cluster in Kubernete
...
 
-    ```shell
-    kubectl apply -f backup-rbac.yaml -n test1
-    ```
+Refer to [Ad-hoc full backup prerequisites](backup-to-aws-s3-using-br.md#prerequisites-for-ad-hoc-full-backup).
 
-2. Create the `s3-secret` secret which stores the credential used to access the S3-compatible storage:
+### Ad-hoc backup process
+
++ Create the `Backup` CR, and back up cluster data by importing AccessKey and SecretKey to grant permissions:
```
Comment (lilin90) [inline en/backup-to-s3.md]:
```suggestion
+ Create the `Backup` CR, and back up cluster data to AWS S3 by importing AccessKey and SecretKey to grant permissions:
```

File: en/backup-to-s3.md [hunk 2]
```diff
@@ -10,113 +10,160 @@ This document describes how to back up the data of the TiDB cluster in Kubernete
...
 
 For the current S3-compatible storage types, Ceph and Amazon S3 work normally as tested. Therefore, this document shows examples in which the data of the `demo1` TiDB cluster in the `test1` Kubernetes namespace is backed up to Ceph and Amazon S3 respectively.
 
-### Prerequisites for ad-hoc backup
+### Three methods to grant permissions of AWS account
 
-1. Download [backup-rbac.yaml](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml) and execute the following command to create the role-based access control (RBAC) resources in the `test1` namespace:
+If you use Amazon S3 to back up and restore the cluster, you have three methods to grant permissions. For details, refer to [Back up TiDB Cluster Data to AWS Using BR](backup-to-aws-s3-using-br.md#three-methods-to-grant-aws-account-permissions). If Ceph is used as backend storage in backup and restore test, the permission is granted by importing AccessKey and SecretKey.
```
Comment (lilin90) [inline en/backup-to-s3.md]:
```suggestion
- If you use Amazon S3 to back up and restore the cluster, you have three methods to grant permissions. For details, refer to [Back up TiDB Cluster Data to AWS Using BR](backup-to-aws-s3-using-br.md#three-methods-to-grant-aws-account-permissions).
- If Ceph is used as backend storage in backup and restore test, the permission is granted by importing AccessKey and SecretKey.
```

File: en/restore-from-s3.md [hunk 3]
```diff
@@ -12,27 +12,50 @@ The restoration method described in this document is implemented based on Custom
 
 This document shows an example in which the backup data stored in the specified path on the S3-compatible storage is restored to the TiDB cluster.
 
-## Prerequisites
+## Three methods to grant AWS account permissions
 
-1. Download [`backup-rbac.yaml`](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml) and execute the following command to create the role-based access control (RBAC) resources in the `test2` namespace:
+If you use Amazon S3 to back up and restore the cluster, you have three methods to grant permissions. For details, refer to [Back up TiDB Cluster Data to AWS Using BR](backup-to-aws-s3-using-br.md#three-methods-to-grant-aws-account-permissions). If Ceph is used as backend storage in backup and restore test, the permission is granted by importing AccessKey and SecretKey.
```
Comment (lilin90) [inline en/restore-from-s3.md]:
```suggestion
- If you use Amazon S3 to back up and restore the cluster, you have three methods to grant permissions. For details, refer to [Back up TiDB Cluster Data to AWS Using BR](backup-to-aws-s3-using-br.md#three-methods-to-grant-aws-account-permissions).
- If Ceph is used as backend storage in backup and restore test, the permission is granted by importing AccessKey and SecretKey.
```

**Comment by ran-huang** [comment]
@lilin90 Comments addressed. PTAL~

**Pull request merged**